sf/draft procurement/stmt of qual.md

```diff
@@ -31,22 +31,26 @@ Play 2 – Address the Whole Experience from Start to Finish
   
 Play 3 – Make it Simple and Intuitive
   * The products or services for our customers/clients do not require knowledge of the organizational structure or our agency to complete the process.
-  * We coordinated with our customers/clients to make design and content decisions based on user research, usability testing, and analytics.
-  * For all user-facing features built over the last quarter, we coordinated with our customers/clients to use plain language that is familiar to the user and easy to understand.
+  * We coordinated with our customers/clients to make design and content decisions based on accessibility needs, user research, usability testing, and analytics.
+  * For all user-facing features built recently, we coordinated with our customers/clients to use plain language that is familiar to the user, easy to understand, and possible to translate across cultures and languages.
   
 Play 4 – Build the Service Using Agile and Iterative Practices
-  * For at least ½ of the projects/products for our customers/clients we launched a functioning “minimum viable product” (MVP) that took no longer than nine(?) months from the beginning of the project, using a “beta” or “test” period if needed.
+  * For at least half of the projects/products for our customers/clients we launched a functioning “minimum viable product” (MVP), using a “beta” or “test” period if needed.
```

Comment:
- Rewrote this item to make it more inclusive, as well as to make it less prescriptive